src/item/VanillaItems.php

```diff
@@ -133,6 +133,7 @@
  * @method static Item DRAGON_BREATH()
  * @method static Skull DRAGON_HEAD()
  * @method static DriedKelp DRIED_KELP()
+ * @method static ItemBlock KELP()
```

Comment:
We don't put blocks in VanillaItems. `VanillaBlocks::KELP()->asItem()` would be used instead.